serrano/resources/preview.py

```diff
@@ -28,13 +27,41 @@ class PreviewResource(BaseResource, PaginatorResource):
 
     parametizer = PreviewParametizer
 
-    def get(self, request):
+    def get(self, request, **kwargs):
         params = self.get_params(request)
 
-        page = params.get('page')
         limit = params.get('limit')
         tree = params.get('tree')
 
+        page = kwargs.get('page')
+        stop_page = kwargs.get('stop_page')
+
+        offset = None
+
+        # Restrict the preview results to a particular page or page range. If
+        # no page range is supplied, use the default limit.
```

Comment:
Do we want this statement to be true or should the limit be `None` when no page is supplied?